libs/coin-framework/src/serialization/operation.ts

```diff
@@ -168,7 +168,10 @@ export const fromOperationRaw = (
   };
 
   if (transactionSequenceNumber !== undefined) {
-    res.transactionSequenceNumber = transactionSequenceNumber;
+    const transactionSequenceNumberBN = new BigNumber(transactionSequenceNumber);
+    res.transactionSequenceNumber = transactionSequenceNumberBN.isNaN()
+      ? undefined
+      : transactionSequenceNumberBN;
```

Comment:
Do we really want an actual `undefined` instead of simply not setting the key ? This is why we used to have this snippet in the first place
```ts
if (transactionSequenceNumber !== undefined) {
    res.transactionSequenceNumber = transactionSequenceNumber;
}
```
```suggestion
    if (!transactionSequenceNumberBN.isNaN()) {
      res.transactionSequenceNumber = transactionSequenceNumberBN;
    }
```